ices/44861.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# https://github.com/rust-lang/rust/issues/44861
+# Playground: https://play.rust-lang.org/?version=nightly&mode=debug&edition=2018&gist=bdd59503a0dc2d33c60a949fc7c57633
+# You need to select `build` instead of `run`.
+rustc --crate-type lib - << END
```

Comment:
This can be `.rs` file by using `crate_type` attribute, I think?

Reply:
Yes, thanks for the suggestion!